Makefile.am

```diff
@@ -5,7 +5,12 @@ EXTRA_DIST = autogen.sh
 ACLOCAL_AMFLAGS = -I build
 
 include tests/names.mk
-test: update-test-suite clean-test-data $(TESTS)
+test:
+	make style unittests integration
+
+	make PYTHONVERSION=2.7 -Cpython-api style unittests integration
```

Comment:
Can you add a "TODO: Use configure to detect Python versions currently installed" here?